packages/modeling/src/curves/bezier/length.js

```diff
@@ -0,0 +1,20 @@
+const lengths = require('./lengths')
+
+/**
+ * Approximates the length of the bezier curve by sampling it at a sequence of points, then adding up all the distances.
+ * This is equivalent to flattening the curve into lines and adding up all the line lengths.
+ *
+ * @example
+ * const b = bezier.create([[0, 0], [0, 10]]);
+ * console.log(length(b)) // output 10
+ * 
+ * @param {Object} bezier a bezier curve.
+ * @param {Number} [segments=100] the number of segments to use when approximating the curve length.
+ * @returns an approximation of the curve's length.
+ * @alias module:modeling/curves/bezier.length
+ */
+const length = (bezier, segments = 100) => {
```

Comment:
wondering if the parameters should be reversed as math/geometry functions always place the object at the end.

the only down side is that the segments would then be required. however, segments is really the driver for the approximate length, and important.

@platypii thoughts?

Reply:
Another option would be to make segments an option inside a an options object the way we do elsewhere. So you could do:
```js
bezier.length({}, b)
bezier.length({ segments: 200 }, b)
```

That would make it future proof in case we want to add some other options later.

Reply:
I thought of this as well, but this `bezier.length({}, b)` just hurts my eyes :).
If you think this is the better option I can make the changes.

Reply:
None of the solutions seem great here. I will defer to @z3dev

Reply:
```
bezier.length(200, b)
bezier.length({ segments: 200 }, b)
```
For length(), the first option is best. There are several geometry functions with the same signature.

```
bezier.arcLengthToT(0.7, b, 100)
bezier.arcLengthToT({arcLength: 0.7, segments: 100}, b)
```
For arcLength(), let's go with the options signature to align with library standards. And of course, some defaults can be supplied for the parameters; arcLength and segments.

However... now there are two 'arcLength' in the call signature. So, i'm wondering if a different name for the function or a different name for the parameter would be in order.

Reply:
If you have any naming suggestions I'm happy to make the changes

Reply:
How about...

`bezier.arcLengthToT({distance: 0.7, segments: 100}, b) // preferred`

if understood correctly, distance is between 0 and L. Is that correct?